utils/redis_helper.py

```diff
@@ -0,0 +1,52 @@
+from django.conf import settings
+from utils.redis_client import RedisClient
+from utils.redis_serializers import DjangoModelSerializer
+
+
+class RedisHelper:
+
+    @classmethod
+    def _load_objects_to_cache(cls, key, objects):
+        conn = RedisClient.get_connection()
+
+        serialized_list = []
+        for obj in objects:
+            serialized_data = DjangoModelSerializer.serialize(obj)
+            serialized_list.append(serialized_data)
+
+        if serialized_list:
+            # *serialized_list means all data in this list
+            conn.rpush(key, *serialized_list)
+            conn.expire(key, settings.REDIS_KEY_EXPIRE_TIME)
+
+
+    @classmethod
+    def load_objects(cls, key, queryset):
+        conn = RedisClient.get_connection()
+        # if there is a cache
+        if conn.exists(key):
+            # get all tweets in this key
+            serialized_list = conn.lrange(key, 0 , -1)
+            objects = []
+            for serialized_data in serialized_list:
+                deserialized_obj = DjangoModelSerializer.deserialize(serialized_data)
+                objects.append(deserialized_obj)
+            return objects
+
+        # if cache miss
+        cls._load_objects_to_cache(key, queryset)
+
+        # 转换为 list 的原因是保持返回类型的统一，因为存在 redis 里的数据是 list 的形式
+        return list(queryset)
+
+
+    @classmethod
+    def push_object(cls, key, obj, queryset):
+        conn = RedisClient.get_connection()
+        if not conn.exists(key):
+            # 如果 key 不存在，直接从数据库里 load
+            # 就不走单个 push 的方式加到 cache 里了
+            cls._load_objects_to_cache(key, queryset)
+            return
+        serialized_data = DjangoModelSerializer.serialize(obj)
+        conn.lpush(key, serialized_data)
```

Comment:
If the key doesn't exist, it will create the key for you and make the value as the set or list depends on the data